src/app/cars/car/car.component.scss

```diff
@@ -1,70 +1,72 @@
-.image {
-    width: 50%;
+.car-card{
     display: flex;
-    justify-content: center;
-    align-items: center;
+    padding: 0;
     overflow: hidden;
-    border-radius: 5px 0 0 5px;
+    min-width: 450px;
```

Comment:
This part of code breaks flexible design (need to support 800px width)

Reply:
[https://ibb.co/ZdSHkpY](url)

Reply:
need to re-check